phpunit.xml.dist

```diff
@@ -18,5 +18,7 @@
     </filter>
     <logging>
         <log type="coverage-clover" target="./clover.xml"/>
+        <log type="coverage-xml" target="coverage/coverage-xml" />
```

Comment:
why do we need 3 different ways of logging the coverage?

Reply:
I'm reusing already generated coverage report in `infection` so it doesn't have to run initial test suite before MTU.

Reply:
sure, but why do you need the 3rd one? :)

Reply:
I think it's using both but I don't know why.

Reply:
ok found it in docs, you can keep it as it is
in the future we can think of integrating with e.g. https://coveralls.io